src/components/asset-search/selected-asset.tsx

```diff
@@ -34,14 +38,15 @@ export const SelectedAsset: React.FC<{ hideArrow?: boolean }> = ({ hideArrow })
   if (!selectedAsset) {
     return null;
   }
-  const { name } = selectedAsset;
+  const { name, contractAddress } = selectedAsset;
+
+  const isStx = name === 'Stacks Token';
+  const ticker = isStx ? 'STX' : getTicker(name).toUpperCase();
```

Comment:
Another place where these names end up with smelly code. Probably the better solution is to have Asset have two interfaces - StxAsset and FungibleAsset, then type Asset = StxAsset | FungibleAsset

Reply:
Also, is there every a reason that `getTicker` should not always return full caps? Then we don't need to `toUpperCase` it always?